vcx/wrappers/java/ci/publishJar.sh

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env bash
 
 set -e
-JAR_FOLDER=vcx/wrappers/java/artifacts/jar
-JAR_VERSION=$(find ${JAR_FOLDER} -type f -name 'com.evernym-vcx_*.jar'| perl -nle 'print $& if m{(?<=vcx_)(.*)(?=.jar)}' | head -1 | awk '{print $1}')
+JAR_FOLDER=artifacts/jar
+JAR_VERSION=$(find ${JAR_FOLDER} -type f -name 'com.evernym-vcx-*.jar'| perl -nle 'print $& if m{(?<=vcx-)(.*)(?=.jar)}' | head -1 | awk '{print $1}')
```

Comment:
Should it be `com.evernym.vcx*` ?

Reply:
I decided to leave the same package name